src/js/components/projects/detail.tsx

```diff
@@ -100,7 +112,8 @@ const ProjectDetail = (props: RouteComponentProps) => {
           label={i18n.t('Submit Project')}
```

Comment:
This should change based on `currentlySubmitting` -- see task detail page for comparison.

Reply:
Also, above `DetailPageLayout` should have `onRenderHeaderActions` passed in with links to view branch/PR. Again, see task detail.